src/components/Registration/index.jsx

```diff
@@ -1,4 +1,81 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import RegistrationForm from './styles';
+
 function Registration() {
-  return <>Ceci est mon formulaire d inscription</>;
+  const [users, setUsers] = useState({
+    email: '',
+    password: '',
+    firstname: '',
+    lastname: '',
+    zipcode: '',
+    country: '',
+    mobile: '',
+    city: '',
+  });
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    axios.post('http://localhost:5050/users', users);
```

Comment:
I'd add a `then()` to notify mu visitor that all went well, but taht's not mandatory